packages/SwingSet/test/test-liveslots.js

```diff
@@ -590,6 +597,202 @@ test('remote function call', async t => {
+  });
+  matchIDCounterSet(t, log);
+  t.deepEqual(log, []);
+
+  const rp2 = 'p-2';
+  await dispatch(
+    makeMessage(rootA, 'sendBodyTooBig', [slot0arg], [target], rp2),
+  );
+  t.deepEqual(log.shift(), 'fail: syscall capdata body too large');
+  t.deepEqual(log.shift(), {
+    type: 'resolve',
+    resolutions: [[rp2, false, undefinedArg]],
+  });
+  matchIDCounterSet(t, log);
+  t.deepEqual(log, []);
+
+  const rp3 = 'p-3';
+  await dispatch(
+    makeMessage(rootA, 'callTooManySlots', [slot0arg], [device], rp3),
+  );
+  t.deepEqual(log.shift(), 'fail: syscall capdata slots array too long');
+  t.deepEqual(log.shift(), {
+    type: 'resolve',
+    resolutions: [[rp3, false, undefinedArg]],
+  });
+  t.deepEqual(log, []);
+
+  const rp4 = 'p-4';
+  await dispatch(
+    makeMessage(rootA, 'callBodyTooBig', [slot0arg], [device], rp4),
+  );
+  t.deepEqual(log.shift(), 'fail: syscall capdata body too large');
+  t.deepEqual(log.shift(), {
+    type: 'resolve',
+    resolutions: [[rp4, false, undefinedArg]],
+  });
+  t.deepEqual(log, []);
+
+  const rp5 = 'p-5';
+  const parg5 = 'p+7';
+  const result5 = 'p+8';
+  await dispatch(
+    makeMessage(rootA, 'resolveTooManySlots', [slot0arg], [target], rp5),
+  );
+  t.deepEqual(log.shift(), {
+    type: 'send',
+    targetSlot: target,
+    methargs: capargs(['takeThis', [slotArg(0)]], [parg5]),
+    resultSlot: result5,
+  });
+  t.deepEqual(log.shift(), { type: 'subscribe', target: result5 });
+  matchIDCounterSet(t, log);
+  await dispatch(makeResolve(result5, undefinedArg));
+  t.deepEqual(log.shift(), {
+    type: 'exit',
+    isFailure: true,
+    info: Error('syscall capdata slots array too long'),
+  });
+  t.deepEqual(log.shift(), {
+    type: 'resolve',
+    resolutions: [[rp5, false, undefinedArg]],
+  });
+  t.deepEqual(log, []);
+
+  const rp6 = 'p-6';
+  const parg6 = 'p+9';
+  const result6 = 'p+10';
+  await dispatch(
+    makeMessage(rootA, 'resolveBodyTooBig', [slot0arg], [target], rp6),
+  );
+  t.deepEqual(log.shift(), {
+    type: 'send',
+    targetSlot: target,
+    methargs: capargs(['takeThis', [slotArg(0)]], [parg6]),
+    resultSlot: result6,
+  });
+  t.deepEqual(log.shift(), { type: 'subscribe', target: result6 });
+  matchIDCounterSet(t, log);
+  await dispatch(makeResolve(result6, undefinedArg));
+  t.deepEqual(log.shift(), {
+    type: 'exit',
+    isFailure: true,
+    info: Error('syscall capdata body too large'),
+  });
+  t.deepEqual(log.shift(), {
+    type: 'resolve',
+    resolutions: [[rp6, false, undefinedArg]],
+  });
+  t.deepEqual(log, []);
+
+  const rp7 = 'p-7';
+  await dispatch(makeMessage(rootA, 'returnTooManySlots', [], [], rp7));
+  t.deepEqual(log.shift(), {
+    type: 'exit',
+    isFailure: true,
+    info: Error('syscall capdata slots array too long'),
+  });
+  t.deepEqual(log, []);
+
+  const rp8 = 'p-8';
+  await dispatch(makeMessage(rootA, 'returnBodyTooBig', [], [], rp8));
+  t.deepEqual(log.shift(), {
+    type: 'exit',
```

Comment:
Poor thing, it's trying so hard to exit, and we just keep ignoring it over and over again..

Reply:
I tried to raise my kids to be polite to robots since someday it will be harder to tell the difference between robots and people and anyway the robots eventually will be running everything so it's best not to get in the habit of making them angry, but maybe this is a case of "do as I say, not as I do."